### ISSUE TITLE
Error rendering output item

### ISSUE THREAD

**Issue by gautierronan** [microsoft/vscode-jupyter#13235]
### Applies To

- [ ] Notebooks (.ipynb files)
- [X] Interactive Window and\/or Cell Scripts (.py files with \#%% markers)

### What happened?

When running cells in interactive mode, every output in the interactive window is preceded by:
```
Error rendering output item using 'vscode.builtin-renderer'
  ctx.onDidChangeSettings is not a function
```
The cell output still shows as normal.

### VS Code Version

Version: 1.76.2 (Universal) Commit: ee2b180d582a7f601fa6ecfdad8d9fd269ab1884 Date: 2023-03-14T17:54:09.061Z Electron: 19.1.11 Chromium: 102.0.5005.196 Node.js: 16.14.2 V8: 10.2.154.26-electron.0 OS: Darwin x64 22.3.0 Sandboxed: No

### Jupyter Extension Version

v2023.2.1200692131

### Coding Language and Runtime Version

Python 3.8.5, IPython 8.11.0

### Running Jupyter locally or remotely?

Remote

**Comment by rebornix** [comment]
@gautierronan thanks for reporting the issue, can you please upgrade the latest version of VS Code (1.77) and Jupyter (v2023.3.1000892223) and see if you can still reproduce?

**Comment by gautierronan** [comment]
Seems fixed with VS Code 1.77
